assets/scripts/admin/tokens.js

```diff
@@ -38,19 +38,19 @@ $.fn.YesNoDate = function(options)
             {
                 // Show date
                 $elDateContainer.show();
-                $elHiddenInput.attr('value', moment().format($elDate.data('date-format')));
+                $elHiddenInput.val(moment().format($elDate.data('date-format')));
```

Comment:
What's the difference?

Reply:
When using val() the value is set in the DOM. When using attr, sometimes, that's not updated on the DOM